introduction.md

```diff
@@ -1,2 +1,8 @@
 mtrojanowski
 my favourite path on CS is Ruby
+
+My deadly skills
+================
+
+* PHP
+* git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?